src/openforms/js/lang/nl.json

```diff
@@ -2622,6 +2627,11 @@
     "description": "JSON editor: object entry has empty key name",
     "originalDefault": "Set a key name before you can configure this variable"
   },
+  "vowYMF": {
+    "defaultMessage": "Weet je zeker dat je het Datatype wilt veranderen?",
+    "description": "Changing user variable data type and transforming initial value confirmation title",
+    "originalDefault": "Are you sure that you want to change the data type?"
+  },
```

Comment:
where is this message coming from? I only see one new `intl.formatMessage` call in this PR

Reply:
Ah yeah, I added this as the title for the new confirmation modal. At this moment this message isn't being used.
I'll remove it